### DISCUSSION TITLE
Changing icons in Ventura ??

### DISCUSSION THREAD

**Discussion by MysterienVril** [elrumo/macOS_Big_Sur_icons_replacements#1432] [General]
I am currently running on Big Sur and I love customizing my icons. I am thinking of upgrading to the Ventura OS so I can get Logic Pro X and Apple does not offer older versions of Logic Pro (which is sad). Does anyone know if customizing icons is still the same in Ventura?

Thanks.

**Comment by ineedpoopoo** [comment]
I'm on the most current version of macOS Ventura and I have no issues changing my icons. They look GREAT!

**Comment by CodeGenStudio** [comment]
I can confirm the same. You can go ahead and change icons just like in Big Sur!
